terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraints/Compare.java

```diff
@@ -79,6 +79,12 @@
     Operator operator();
 
     /**
+     * @return {@code true} it's valid if left and right are both null, but it's invalid if either is null. otherwise it's valid

```

Comment:
- [ ] @return configure how to treat {@code null} value in given parameters. If true, it is regarded as valid only when both left and right are {@code null}. It is regarded as invalid when either is {@code null}. If false, both cases are regarded as valid.